### ISSUE TITLE
Entity: insertIgnore and insertManyIgnore methods

### ISSUE THREAD

**Issue by xvs32x** [ngrx/platform#1304]
## Describe any alternatives/workarounds you're currently using
No

## Other information:
Methods ignore existed things in collection.
## If accepted, I would be willing to submit a PR for this feature
Yes

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[x ] No

**Comment by timdeschryver** [comment]
Could you elaborate a bit more please?
Not to be harsh, but I have no idea what you're asking.

**Comment by xvs32x** [comment]
Yes. The entity adapter contains methods for manipulation with collection (addOne,  addMany, etc...)
However there are no methods which ignored the existing elements and only added new elements, automatically defining it. It should work like INSERT IGNORE method in mysql
